src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/GenericAsyncCollectorBindingProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    // Bind Attribute --> IAsyncCollector<TMessage>
+    internal class GenericAsyncCollectorBindingProvider<TAttribute, TMessage> :
```

Comment:
I was looking for something to say "these are shared across many bindings"; in contrast to many of the existing BindingProviders which tend to be hard-wired to a specific binding.    They're internal so we're flexible here. Do you have a preference? Generic? Common? Shared?

---
In reply to: [61139432](https://github.com/Azure/azure-webjobs-sdk/pull/694#discussion_r61139432) [](ancestors = 61139432)